spec/models/court_spec.rb

```diff
@@ -20,6 +20,14 @@
     }
   }
 
+  let(:gbs) { { 'gbs' => 'X123' } }
```

Comment:
Now that we are using the same approach as the email, we can move the `gbs` key-value to the `let(:data)` and remove the allow_any_instance_of, in favour of testing the behaviour, in a similar context as the one we have for the email, `context 'email'`